src/SFA.DAS.Reservations.Domain/Entities/AccountLegalEntity.cs

```diff
@@ -14,7 +14,6 @@ public class AccountLegalEntity
         public int? ReservationLimit { get; set; }
         public bool AgreementSigned { get; set; }
         public bool IsLevy { get; set; }
-        public AgreementType AgreementType { get; set; }
```

Comment:
Can you change the indexes on the table as well as the table configuration - will have a further chat to make sure its ok to remove the data.